src/index.js

```diff
@@ -1,33 +1,73 @@
 'use strict';
 
-const $tagsInput = document.getElementById('tags-input');
-const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
+const $tagsInputFirst = document.getElementById('tags-input-1');
+const $tagsInputSecond = document.getElementById('tags-input-2');
+const $tagsList = document.getElementsByClassName('tags-list');
+const $modal = document.getElementById('question'); // modal window
+const $btnsInModal = $modal.getElementsByClassName('btn');  // buttons Remove and Close in modal window
+const $btnCloseInModal = document.getElementById('close-btn');  // span-button to close modal window
+const tagsArrFirst = [];  // array for first input
+const tagsArrSecond = ['tag1', 'some awesome tag'];   // array for second input
 
-$tagsInput.addEventListener('keyup', ({key, target}) => {
+$tagsInputFirst.addEventListener('keyup', ({key, target}) => {  // event for first input
```

Comment:
you have two almost equal code snippets for "$tagsInputFirst" and "$tagsInputSecond" listeners
better to move callback to some function
👎